assets/styles/nav.scss

```diff
@@ -0,0 +1,362 @@
+/*////// Main bar //////*/
+
+header {
+    height: 69px;
```

Comment:
please use className or else all header tags will take this behavior (or if it is a will, put it on app.scss)